.github/pull_request_template.md

```diff
@@ -9,7 +9,79 @@ Closes #
+
+    - ``changes/<PR#>.general.rst``: infrastructure or miscellaneous change
+    - ``changes/<PR#>.docs.rst``
+    - ``changes/<PR#>.stpipe.rst``
+    - ``changes/<PR#>.datamodels.rst``
+    - ``changes/<PR#>.scripts.rst``
+    - ``changes/<PR#>.fits_generator.rst``
+    - ``changes/<PR#>.set_telescope_pointing.rst``
+
+    ## steps
+    - ``changes/<PR#>.align_refs.rst``
+    - ``changes/<PR#>.ami.rst``
+    - ``changes/<PR#>.assign_mtwcs.rst``
+    - ``changes/<PR#>.assign_wcs.rst``
+    - ``changes/<PR#>.background.rst``
+    - ``changes/<PR#>.barshadow.rst``
+    - ``changes/<PR#>.charge_migration.rst``
+    - ``changes/<PR#>.combine_1d.rst``
+    - ``changes/<PR#>.cube_build.rst``
+    - ``changes/<PR#>.cube_skymatch.rst``
+    - ``changes/<PR#>.dark_current.rst``
+    - ``changes/<PR#>.dq_init.rst``
+    - ``changes/<PR#>.emicorr.rst``
+    - ``changes/<PR#>.engdb_tools.rst``
+    - ``changes/<PR#>.exp_to_source.rst``
+    - ``changes/<PR#>.extract_1d.rst``
+    - ``changes/<PR#>.extract_2d.rst``
+    - ``changes/<PR#>.firstframe.rst``
+    - ``changes/<PR#>.flatfield.rst``
+    - ``changes/<PR#>.fringe.rst``
+    - ``changes/<PR#>.gain_scale.rst``
+    - ``changes/<PR#>.group_scale.rst``
+    - ``changes/<PR#>.guider_cds.rst``
+    - ``changes/<PR#>.imprint.rst``
+    - ``changes/<PR#>.ipc.rst``
+    - ``changes/<PR#>.jump.rst``
+    - ``changes/<PR#>.klip.rst``
+    - ``changes/<PR#>.lastframe.rst``
+    - ``changes/<PR#>.linearity.rst``
+    - ``changes/<PR#>.master_background.rst``
+    - ``changes/<PR#>.mrs_imatch.rst``
+    - ``changes/<PR#>.msaflagopen.rst``
+    - ``changes/<PR#>.nsclean.rst``
+    - ``changes/<PR#>.outlier_detection.rst``
+    - ``changes/<PR#>.pathloss.rst``
+    - ``changes/<PR#>.persistence.rst``
+    - ``changes/<PR#>.photom.rst``
+    - ``changes/<PR#>.pixel_replace.rst``
+    - ``changes/<PR#>.ramp_fitting.rst``
+    - ``changes/<PR#>.refpix.rst``
+    - ``changes/<PR#>.resample.rst``
+    - ``changes/<PR#>.resample_spec.rst``
+    - ``changes/<PR#>.reset.rst``
+    - ``changes/<PR#>.residual_fringe.rst``
+    - ``changes/<PR#>.rscd.rst``
+    - ``changes/<PR#>.saturation.rst``
+    - ``changes/<PR#>.skymatch.rst``
+    - ``changes/<PR#>.source_catalog.rst``
+    - ``changes/<PR#>.spectral_leak.rst``
+    - ``changes/<PR#>.srctype.rst``
+    - ``changes/<PR#>.stack_refs.rst``
+    - ``changes/<PR#>.straylight.rst``
+    - ``changes/<PR#>.superbias.rst``
+    - ``changes/<PR#>.timeconversion.rst``
+    - ``changes/<PR#>.transforms.rst``
+    - ``changes/<PR#>.tso_photometry.rst``
+    - ``changes/<PR#>.tweakreg.rst``
+    - ``changes/<PR#>.wavecorr.rst``
+    - ``changes/<PR#>.wfs_combine.rst``
+    - ``changes/<PR#>.wfss_contam.rst``
+    - ``changes/<PR#>.white_light.rst``
+    </details>
 - [ ] updated or added relevant tests
 - [ ] updated relevant documentation
 - [ ] added relevant milestone
```

Comment:
**Checklist for PR authors (skip items if you don't have permissions or they are not applicable)**
 - [ ] for a public change, added a towncrier news fragment in `changes/` <details><summary>`echo "changed something" > changes/<PR#>.<changetype>.rst`</summary>

     - ``changes/<PR#>.general.rst``: infrastructure or miscellaneous change
     - ``changes/<PR#>.docs.rst``
     - ``changes/<PR#>.stpipe.rst``
     - ``changes/<PR#>.datamodels.rst``
     - ``changes/<PR#>.scripts.rst``
     - ``changes/<PR#>.fits_generator.rst``
     - ``changes/<PR#>.set_telescope_pointing.rst``

     ## steps
     - ``changes/<PR#>.align_refs.rst``
     - ``changes/<PR#>.ami.rst``
     - ``changes/<PR#>.assign_mtwcs.rst``
     - ``changes/<PR#>.assign_wcs.rst``
     - ``changes/<PR#>.background.rst``
     - ``changes/<PR#>.barshadow.rst``
     - ``changes/<PR#>.charge_migration.rst``
     - ``changes/<PR#>.combine_1d.rst``
     - ``changes/<PR#>.cube_build.rst``
     - ``changes/<PR#>.cube_skymatch.rst``
     - ``changes/<PR#>.dark_current.rst``
     - ``changes/<PR#>.dq_init.rst``
     - ``changes/<PR#>.emicorr.rst``
     - ``changes/<PR#>.engdb_tools.rst``
     - ``changes/<PR#>.exp_to_source.rst``
     - ``changes/<PR#>.extract_1d.rst``
     - ``changes/<PR#>.extract_2d.rst``
     - ``changes/<PR#>.firstframe.rst``
     - ``changes/<PR#>.flatfield.rst``
     - ``changes/<PR#>.fringe.rst``
     - ``changes/<PR#>.gain_scale.rst``
     - ``changes/<PR#>.group_scale.rst``
     - ``changes/<PR#>.guider_cds.rst``
     - ``changes/<PR#>.imprint.rst``
     - ``changes/<PR#>.ipc.rst``
     - ``changes/<PR#>.jump.rst``
     - ``changes/<PR#>.klip.rst``
     - ``changes/<PR#>.lastframe.rst``
     - ``changes/<PR#>.linearity.rst``
     - ``changes/<PR#>.master_background.rst``
     - ``changes/<PR#>.mrs_imatch.rst``
     - ``changes/<PR#>.msaflagopen.rst``
     - ``changes/<PR#>.nsclean.rst``
     - ``changes/<PR#>.outlier_detection.rst``
     - ``changes/<PR#>.pathloss.rst``
     - ``changes/<PR#>.persistence.rst``
     - ``changes/<PR#>.photom.rst``
     - ``changes/<PR#>.pixel_replace.rst``
     - ``changes/<PR#>.ramp_fitting.rst``
     - ``changes/<PR#>.refpix.rst``
     - ``changes/<PR#>.resample.rst``
     - ``changes/<PR#>.resample_spec.rst``
     - ``changes/<PR#>.reset.rst``
     - ``changes/<PR#>.residual_fringe.rst``
     - ``changes/<PR#>.rscd.rst``
     - ``changes/<PR#>.saturation.rst``
     - ``changes/<PR#>.skymatch.rst``
     - ``changes/<PR#>.source_catalog.rst``
     - ``changes/<PR#>.spectral_leak.rst``
     - ``changes/<PR#>.srctype.rst``
     - ``changes/<PR#>.stack_refs.rst``
     - ``changes/<PR#>.straylight.rst``
     - ``changes/<PR#>.superbias.rst``
     - ``changes/<PR#>.timeconversion.rst``
     - ``changes/<PR#>.transforms.rst``
     - ``changes/<PR#>.tso_photometry.rst``
     - ``changes/<PR#>.tweakreg.rst``
     - ``changes/<PR#>.wavecorr.rst``
     - ``changes/<PR#>.wfs_combine.rst``
     - ``changes/<PR#>.wfss_contam.rst``
     - ``changes/<PR#>.white_light.rst``
     </details>
 - [ ] updated or added relevant tests
 - [ ] updated relevant documentation
 - [ ] added relevant milestone